integration/helpers/create-fixture.tsx

```diff
@@ -23,6 +23,7 @@ export type Fixture = Awaited<ReturnType<typeof createFixture>>;
 export type AppFixture = Awaited<ReturnType<typeof createAppFixture>>;
 
 export const js = String.raw;
+export const css = String.raw;
```

Comment:
Technically not needed but it helps w/ syntax highlighting plugins that support CSS-in-JS